src/hepattn/models/encoder.py

```diff
@@ -97,6 +97,8 @@ def forward(self, x: Tensor, **kwargs) -> Tensor:
             return x + self.dp(self.ls(self.fn(x, **kwargs)))
         if self.kv_norm and "kv" in kwargs:  # TODO: require kv norm if doing cross attention
             kwargs["kv"] = self.kv_norm(kwargs["kv"])
+            if "v" in kwargs:
+                kwargs["v"] = self.kv_norm(kwargs["v"])
```

Comment:
let's revert this, I don't think we want to reuse the same norm. `kv_norm` is not really used anywhere atm